deltachat-ios/AppDelegate.swift

```diff
@@ -51,6 +52,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
         let console = ConsoleDestination()
         logger.addDestination(console)
+        dcContext.logger = DcLogger()
```

Comment:
We're still having a global `logger` in `AppDelegate.swift line 9` which we are using a lot.

Reply:
That's right. I didn't want to have SwiftyBeaver as a depenency in DcCore. That's why I've defined a Logger interface in DcCore. The main module implements that Logger interface in the class DcLogger and delegates all calls to the global Swifty beaver instance (the `logger` you mentioned).